### PR TITLE
[CI:DOCS] Update a way out of date transfer document

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@TomSweeneyRedHat PTAL

File: transfer.md [hunk 1]
```diff
@@ -63,54 +69,64 @@ There are other equivalents for these tools
...
 
 ## Missing commands in podman
 
 Those Docker commands currently do not have equivalents in `podman`:
 
 | Missing command | Description|
 | :--- | :--- |
-| `docker container rename`   | podman does not support `container rename` - or the `rename` shorthand. We recommend using `podman rm` and  `podman create` to create a container with a specific name.|
+| `docker builder`  ||
+| `docker build*`   ||
```
Comment (mheon) [inline transfer.md:120]: I think we have this one?

Comment (mheon) [reply transfer.md:120]: Did you mean `buildx`?

Comment (rhatdan) [reply transfer.md]: Yup, docker --help refers to this as buildx*, some how we dropped the x, now I am dropping the *

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: transfer.md [hunk 2]
```diff
@@ -63,54 +69,64 @@ There are other equivalents for these tools
 | `docker ps`      | [`podman ps`](./docs/source/markdown/podman-ps.1.md)            |
 | `docker pull`    | [`podman pull`](./docs/source/markdown/podman-pull.1.md)        |
 | `docker push`    | [`podman push`](./docs/source/markdown/podman-push.1.md)        |
+| `docker rename`  | [`podman rename`](./docs/source/markdown/podman-restart.1.md)   |
```
Comment (Luap99) [inline transfer.md:72]: wrong link

File: transfer.md [hunk 3]
```diff
@@ -21,39 +21,45 @@ As well as some systemd helpers like `systemd-cgls` and `systemd-cgtop` are stil
...
 | `docker logout`  | [`podman logout`](./docs/source/markdown/podman-logout.1.md)    |
+| `docker logs` | [`podman logs`](./docs/source/markdown/podman-logs.1.md)                 |
+| `docker manifest         `| [`podman manifest`](./docs.source/markdown/podman-manifest.1.md)           |
+| `docker manifest annotate`  | [`podman manifest annotate`](./docs/source/markdown/podman-manifest-annotate.1.md)   |
+| `docker manifest create` | [`podman manifest create`](./docs/source/markdown/podman-manifest-create.1.md)   |
+| `docker manifest inspect`| [`podman manifest inspect`](./docs/source/markdown/podman-manifest-inspect.1.md) |
+| `docker manifest push`   | [`podman manifest push`](./docs/source/markdown/podman-manifest-push.1.md)   |
+| `docker manifest rm`     | [`podman manifest rm`](./docs.source/markdown/podman-manifest-rm.1.md)           |
+| `docker network `        | [`podman network`](./docs.source/markdown/podman-network.1.md)           |
+| `docker network connect` | [`podman network connect`](./docs/source/markdown/podman-network-connect.1.md)   |
```
Comment (Luap99) [inline transfer.md:62]: network disconnect is missing here

Comment (rhatdan) [reply transfer.md:61]: added

File: transfer.md (hunk 2 above)
Comment (rhatdan) [reply transfer.md]: fixed

**Pull request merged**